main/context_processors.py

```diff
@@ -0,0 +1,21 @@
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+def navbar_extras(request):
```

Comment:
Why is it necessary to filler out the authenticated user from the models?
Is it possible to use the logged in user in this context?